docs/usage/delete.rst

```diff
@@ -7,6 +7,12 @@ Examples
     # delete a single backup archive:
     $ borg delete /path/to/repo::Monday
 
+    # delete all archives whose names begin with "T"
+    $ borg delete --prefix T /path/to/repo
+
+    # delete all archives whose names contain "day"
+    $ borg delete --glob-archives *day* /path/to/repo
```

Comment:
if you happen to have some filename in the current directory that matches `*day*`, the shell will already expand that pattern and borg won't see it like you intended. thus, you need to use `'*day*'` here.

Reply:
just noticed that it might be nice to show how to delete archives from some specific old year, like with `*-2012-*`.

maybe that is a more practical example (besides one would usually delete old stuff by prune)?